rust/src/detect/mod.rs

```diff
@@ -105,6 +105,44 @@ extern {
     ) -> *mut c_void;
 }
 
+// needed for calls to DetectAppLayerMultiRegister
+pub const SIG_FLAG_TOSERVER: u32 = 0x80000; // BIT_U32(19)
```

Comment:
this is an accident waiting to happen, can we get to a single place to define it?

Reply:
Removing the need for it in rust, by using a helper function as for regular single buffers